tests/beman/inplace_vector/spec.test.cpp

```diff
@@ -0,0 +1,231 @@
+#include "beman/inplace_vector/inplace_vector.hpp"
+#include "gtest/gtest.h"
+#include <gtest/gtest.h>
+#include <type_traits>
+
+using namespace beman;
+
+// 23.3.14.1 Overview [inplace.vector.overview]
+// 1 An inplace_vector is a contiguous container. Its capacity is fixed and its
+// elements are stored within the inplace_vector object itself.
+//
+// This is not testable.
+
+// 2 An inplace_vector meets all of the requirements of a container (23.2.2.2),
+// of a reversible container (23.2.2.3), of a contiguous container, and of a
+// sequence container, including most of the optional sequence container
+// requirements (23.2.4). The exceptions are the push_front, prepend_range,
+// pop_front, and emplace_front member functions, which are not provided.
+// Descriptions are provided here only for operations on inplace_- vector that
+// are not described in one of these tables or for operations where there is
+// additional semantic information.
+//
+// TODO: Test container & reversible container & optional sequence container
+// req.
+
```

Comment:
I will drop a suggestion for 23.2.2.2 soon.

Reply:
Seems I still cannot push anything to this branch. Can anyone help to enable this?

Reply:
Let me add you write premission to this repo

Reply:
I have sent you an invite. Note this does not grant you write access to other beman repos, if you wish to have the privilege of general write access, please contact leads on our discourse.